src/h5web/visualizations/line/MappedLineVis.tsx

```diff
@@ -8,12 +8,14 @@ import { useLineConfig } from './config';
 
 interface Props {
   value: HDF5Value;
-  dataset: HDF5Dataset;
+  dims: number[];
   mapperState: DimensionMapping;
+  axesLabels?: (string | undefined)[];
+  axesValues?: Record<string, HDF5Value>;
 }
 
 function MappedLineVis(props: Props): ReactElement {
-  const { value, dataset, mapperState } = props;
+  const { value, axesLabels, axesValues = {}, dims, mapperState } = props;
```

Comment:
With a default empty array for `axesLabels`, you can remove `axesLabels &&` below.